infra/style/format.sh

```diff
@@ -6,13 +6,42 @@ set -e
 APPLY_PATCH_OPTION="-a"
 while [[ "$#" -gt 0 ]]; do
   case $1 in
-    --no-apply-patches) APPLY_PATCH_OPTION="";;
+    --no-apply-patches)
+      APPLY_PATCH_OPTION=""
+      shift
+      ;;
+    --diff-only)
```

Comment:
How about providing short option together, like `-d`?

Reply:
> Or you can make diff-only as default.

I've thought about this idea, but it has some side effects to other codes. We may need some discussion before doing that.